recipes/dfast/meta.yaml

```diff
@@ -3,29 +3,24 @@ package:
   version: 1.2.3
 
 build:
-  number: 0
-  noarch: generic
+  number: 1
+  noarch: python
 
 source:
   url: https://github.com/nigyta/dfast_core/archive/1.2.3.tar.gz
   sha256: 710d2cad3874aba691e5995bc3865785e1807a09bf3a8a8c622126d7d4551859 
 
 requirements:
-  build:
-    - python >=3.5
+  host:
+    - python >=3.6
     - biopython
```

Comment:
You can actually remove the host section, nothing in here is used during installation.